packages/flame/lib/src/effects/controllers/noise_effect_controller.dart

```diff
@@ -21,7 +18,9 @@ import 'package:vector_math/vector_math_64.dart';
 /// example, putting into a `MoveEffect.by` will create a shake motion, where
 /// the magnitude and the direction of shaking is controlled by the effect's
 /// `offset`.
+@Deprecated('Use flame_noise instead')
 class NoiseEffectController extends DurationEffectController {
+  @Deprecated('Use flame_noise instead')
```

Comment:
I don't think you have to deprecate both the constructor and the class? I think the class should be enough.

Reply:
I recall vscode complaining if the constructor wasn't. Let me try again

Reply:
![image](https://user-images.githubusercontent.com/882703/224895037-97ee881d-1175-495a-8ddc-2cff9eb45c76.png)